R/nice_fit.R

```diff
@@ -38,6 +38,12 @@ nice_fit <- function(..., model.labels, nice_table = FALSE) {
   df <- do.call(rbind, x)
   if (!missing(model.labels)) {
     Model <- model.labels
+    # verify labels match number of objects
+    if(!length(x) == length(model.labels)){
+      warning("Number of model.labels and models do not match. Broadcasting...")
```

Comment:
I like that it is a warning instead of an error because then it still works if you provide it with a single vector (if for some weird reason you'd like to have the same name for all models), but then it will naturally generate an error if it's too big and also add your warning.

Reply:
Great catch, I will check this with a Unit Test if this causes a failure and then include a custom error message.

Reply:
Think this should be resolved with the latest commits?